### ISSUE TITLE
[Request] Create pytorch+fastai lambda layer

### ISSUE THREAD

**Issue by nicolasmetallo** [mattmcclean/pytorch-lambda-layer#1]
Hi Matt,

Thank you very much for putting together this repo. It's great to see you were able to get PyTorch 1.2 on Lambda. How much space is it taking from `/tmp`? On my tests, it was around 400MB (more than version 1.1.0) and was very limiting.

Using 1.1.0 and a slim version from fastai, I was able to get a text classification model to work on Lambda (serialisation is not available for trained language models ie TransformerXL). I wasn't able to get `load_learner` to work as there's no shared memory space, but loading `data_clas` into a new `text_classification_learner` did the trick. Normally you would pass the arguments `bs=1` and `num_workers=0` to `load_learner` to fix this but they are being overwritten for some reason. **Have you tried using fastai on Lambda?**

To get fastai to work, I created a zip with the dependencies and then I stream that to memory during warm-up, unzipping into `/tmp`. I run:
```
RUN pip install --no-deps fastai
RUN git clone https://github.com/fastai/fastai
RUN cd fastai && tools/run-after-git-clone
RUN cd fastai && pip install $(python setup.py -q deps --dep-groups=core,text)
```
Remove `torchvision` and extra languages we don't need from `spaCy`:
```
RUN cd /deps/spacy/lang && find . -maxdepth 1 -type d -not -name es -not -name "." -exec rm -ir -rf {} \;
```

Thank you very much for your work.
Nico
